js/console_test.go

```diff
@@ -217,7 +219,7 @@ func TestConsoleLog(t *testing.T) {
 
 			vu := initVU.Activate(&lib.VUActivationParams{RunContext: ctx})
 
-			logger := extractLogger(vu.(*ActiveVU).Console.logger)
+			logger := extractLogger(vu.(*ActiveVU).Console.logger) //nolint:forcetypeassert
```

Comment:
The `extractLogger` signature could be easily changed where it accepts the VU and there we can just in case do the assertion.

Reply:
[ca79957](https://github.com/grafana/k6/pull/3393/commits/ca79957168d32aa960b308fcdfbf2aff956cf7d7)